doc-content/enterprise-only/release-notes/rn-known-issues-ref.adoc

```diff
@@ -3,106 +3,215 @@
 
-.In guided rules, guided decision tables, and guided decision templates, the use of the "contains" operator does not add values from enumerations that have quotes. [https://issues.jboss.org/browse/RHDM-25[RHDM-25]]
+.The controller URL is not updated in the standalone.xml file if only {KIE_SERVER} is installed [https://issues.jboss.org/browse/RHPAM-1781[RHPAM-1781]]
 
-Issue: When enumeration is used for a string field of a fact in guided rules, guided decision tables, and guided decision templates, the resultant rule does not wrap the selected value from the enumeration drop down in quotes ("") and a validation error occurs. 
-
-ifdef::PAM[]
-.Process designer instances are very slow with Google Chrome on Linux [https://issues.jboss.org/browse/RHPAM-922[RHPAM-922]]
+Issue: When you run the {PRODUCT} installer and you specify a URL for the controller, the URL is not updated in the {EAP} `standalone.xml` and `standalone-full.xml` files.
 
-Step to reproduce: Open three process designer instances in Google Chrome on Linux.
+Steps to reproduce:
 
-Expected result: All instances of process designer perform at the same level, regardless of the number of instances open.
+. Run the {PRODUCT} installer.
+. Specify an {EAP} installation for container.
+. Select only Process Server to install.
+. Select Advanced Configuration and update controller URL.
+. Complete the installation.
+. Open the {EAP} `standalone.xml` or  `standalone-full.xml` file and find the controller URL.
 
-Actual result: The first two instances that were opened perform as usual, but the third and all subsequent instances are very slow, even if you close the first two instances.
+Expected result: The value of the controller URL property is the URL that you entered during installation.
 
-Workaround: Set the value of `chrome://flags/#force-color-profile` to sRGB.
+Actual result: The value of the controller URL property is the default value.
 
-endif::[]
+Workaround: Manually update the value of the controller URL property in the {EAP} `standalone.xml` and  `standalone-full.xml` file.
 
-.Project cannot be built and deployed in {CENTRAL} when connected to a snapshot Maven repo [https://issues.jboss.org/browse/RHPAM-1544[RHPAM-1544]]
+== {CENTRAL}
+.Red Hat Decision Central 7.0 and earlier and BRMS 6.4 and earlier repositories are not fulled migrated when imported into {CENTRAL} [https://issues.jboss.org/browse/RHPAM-1768[RHPAM-1768]]
 
+Issue: Red Hat Decision Central 7.0 and earlier and BRMS 6.4 and earlier repositories (old structure repositories) that contain more than one project can be imported into {CENTRAL} even though they are not supported. When these repositories are imported, the are not fully migrated and will not perform as expected.
```

Comment:
"repositories (with the legacy repository structure)" or something other than "old structure repositories"